src/plugins/footnotes/index.ts

```diff
@@ -203,13 +214,32 @@ export default () => {
                 )
               )
             }
+
+            if (
+              tableInlineFootnoteIds &&
```

Comment:
updated if to be undefined in case footnotes are not in table wrap footnote

Reply:
what about `tableInlineFootnoteIds`?

Reply:
this will defined in case footnotes are parent to a `table_element_footer`, and will be undefined if those footnotes are not in `table_element_footer`